packages/lesswrong/components/hooks/useRecordPostView.tsx

```diff
@@ -32,7 +32,7 @@ export const useRecordPostView = (post: ViewablePost): {recordPostView: any, isR
   const {recordEvent} = useNewEvents()
   const currentUser = useCurrentUser();
   const {postsRead, setPostRead} = useItemsRead();
-  const isRead = post && ((post._id in postsRead) ? postsRead[post._id] : post.isRead)
+  const isRead = post && ((post._id in postsRead) ? postsRead[post._id] : post.isRead ?? false)
```

Comment:
I'd probably just say `post && !!((post._id in postsRead) ? postsRead[post._id] : post.isRead)` instead, which is one less level of nested logic (sort of).